src/library/basetracktablemodel.cpp

```diff
@@ -612,7 +618,10 @@ QVariant BaseTrackTableModel::roleValue(
             }
             bool ok;
             const auto timesPlayed = rawValue.toInt(&ok);
-            VERIFY_OR_DEBUG_ASSERT(ok && timesPlayed >= 0) {
+            VERIFY_OR_DEBUG_ASSERT(ok) {
```

Comment:
Do we need to split all such debug assertions now? In this case using `&` in the condition was intended.

Reply:
Without the split, the link assertion kicks in. I have not jet an explanation why.
I assume that the compiler optimizes it in a way that it creates a two separate branched where one has no longer a good path. 

That the original code no longer works is really a caveat of this solution.

I think I need to examine the ASM code to find out what is going on.